### ISSUE TITLE
as a developer I want to measure the test coverage after alpha0

### ISSUE THREAD

**Issue by dpriskorn** [internetarchive/wcdimportbot#500]

**Comment by dpriskorn** [comment]
Name                                                           Stmts   Miss  Cover
----------------------------------------------------------------------------------
config.py                                                         39      0   100%
src/__init__.py                                                   60     21    65%
src/helpers/__init__.py                                            0      0   100%
src/helpers/console.py                                             2      0   100%
src/models/__init__.py                                             0      0   100%
src/models/api/__init__.py                                        17      7    59%
src/models/api/enums.py                                            5      0   100%
src/models/api/get_article_statistics.py                          57     40    30%
src/models/api/get_statistics_schema.py                           10      1    90%
src/models/api/job.py                                              6      0   100%
src/models/api/lookup_by_wikidata_qid.py                          29     17    41%
src/models/api/lookup_wikicitations_qid.py                        43     27    37%
src/models/exceptions.py                                          22      0   100%
src/models/person/__init__.py                                     28     10    64%
src/models/return_/__init__.py                                     4      0   100%
src/models/return_/wikibase_return.py                              7      0   100%
src/models/wcd_item.py                                            10      0   100%
src/models/wikibase/__init__.py                                  140     30    79%
src/models/wikibase/crud/__init__.py                               0      0   100%
src/models/wikibase/dictionaries.py                                9      0   100%
src/models/wikibase/enums.py                                      16      0   100%
src/models/wikibase/ia_sandbox_wikibase.py                        74      0   100%
src/models/wikibase/wikicitations_wikibase.py                     73      0   100%
src/models/wikimedia/__init__.py                                   0      0   100%
src/models/wikimedia/enums.py                                     12      0   100%
src/models/wikimedia/wikipedia/__init__.py                         0      0   100%
src/models/wikimedia/wikipedia/article.py                        142     91    36%
src/models/wikimedia/wikipedia/reference/__init__.py               0      0   100%
src/models/wikimedia/wikipedia/reference/english/__init__.py       0      0   100%
src/models/wikimedia/wikipedia/reference/enums.py                 10      0   100%
src/models/wikimedia/wikipedia/reference/extractor.py            112     39    65%
src/models/wikimedia/wikipedia/reference/generic.py              666    290    56%
src/models/wikimedia/wikipedia/reference/raw_reference.py        141     94    33%
src/models/wikimedia/wikipedia/reference/template.py              98     70    29%
src/wcd_base_model.py                                             16      7    56%
test_data/test_content.py                                          2      0   100%
----------------------------------------------------------------------------------
TOTAL                                                           1850    744    60%

**Comment by ollie-iterators** [comment]
https://github.com/internetarchive/wcdimportbot/blob/main/TEST_COVERAGE.txt has different levels of test coverage

**Comment by dpriskorn** [comment]
Thanks for the heads up. I'll create a new helper script that update that file automatically.

**Comment by ollie-iterators** [comment]
I noticed today that the Pull Request: https://github.com/internetarchive/wcdimportbot/pull/534 added some tests and that the test coverage file: https://github.com/internetarchive/wcdimportbot/blob/main/TEST_COVERAGE.txt was not updated. Why is this?

**Comment by dpriskorn** [comment]
It is not done automatically.

**Comment by ollie-iterators** [comment]
Ok. I think it would be nice if it was

**Comment by dpriskorn** [comment]
I'm running the update script before every release as part if the release process.

**Comment by ollie-iterators** [comment]
Version 3.0.0-alpha2 was released and the test coverage did not change.

**Comment by dpriskorn** [comment]
Correct. Actually a rather large share of the tests (31) do not pass. 
Next sprint is a rewrite sprint and after that is a maintenance sprint with focus on testing and coverage.
